crypto/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 #[cfg(not(feature = "std"))]
+#[macro_use]
```

Comment:
was this because `vec!` was missing? and maybe `format!` as well

Reply:
Yeah.